### PR TITLE
Create BlueFrontPixel/RedFrontPixel, added reset feat

### PR DESCRIPTION
**Pull request by gwarby**
Feat: for arm command functions, if movement of 0 is input, arm will set target position to 0. Ex: armextend(0, 0.3); arm will go to 0 tick position (resetting)

### CODE REVIEW

File: BlueFrontPixel.java [hunk 1]
```diff
@@ -0,0 +1,613 @@
...
+import org.opencv.imgproc.Imgproc;
+import org.openftc.easyopencv.OpenCvCamera;
+import org.openftc.easyopencv.OpenCvCameraFactory;
+import org.openftc.easyopencv.OpenCvCameraRotation;
+import org.openftc.easyopencv.OpenCvPipeline;
+import org.openftc.easyopencv.OpenCvWebcam;
+
+
+@Autonomous(name = "BlueFrontPixel")
+public class RedBackPixel extends LinearOpMode
```
Comment (lcdnbl) [inline BlueFrontPixel.java]: Line 38, RedBack -> BlueFront

File: BlueFrontPixel.java [hunk 2]
```diff
@@ -0,0 +1,613 @@
...
+    if (opModeIsActive()) {
+      /************************************************************************
+       * START / RUN OPMODE CODE:
+       *   -perform autonomous driving based on identified prop location
+       ************************************************************************/
+      
+      if (findPropPL.propLocation == "LEFT"){
+        /************************************************************************
+         *
+         * LEFT: (RED BACK)
```
Comment (lcdnbl) [inline BlueFrontPixel.java]: Line 169, RED BACK -> BLUE FRONT

File: BlueFrontPixel.java [hunk 3]
```diff
@@ -0,0 +1,613 @@
...
+      drive(60, 0, 0, DRIVE_POWER);      // Drive forward to parking zone, with second pixel
+      openClamp();                      // Drop pixel to score
+      armextend(0, 0.3);
+      armraise(0, 0.159);     // finish lowering claw to ground
+      normalFlipper();            // square w/ ground
+
+      } else if (findPropPL.propLocation == "MIDDLE") { // If pixel is in MIDDLE
+        /************************************************************************
+         *
+         * MIDDLE: (RED BACK)
```
Comment (lcdnbl) [inline BlueFrontPixel.java]: Line 203, same

File: BlueFrontPixel.java [hunk 4]
```diff
@@ -0,0 +1,613 @@
...
+      drive(67, 0, 0, DRIVE_POWER);      // Drive forward to parking zone, with second pixel
+      openClamp();                      // Drop pixel to score
+      armextend(0, 0.3);
+      armraise(0, 0.159);     // finish lowering claw to ground
+      normalFlipper();            // square w/ ground
+
+      } else { // RIGHT code
+        /************************************************************************
+         *
+         * RIGHT: (RED BACK)
```
Comment (lcdnbl) [inline BlueFrontPixel.java]: Line 234, same

**Review by lcdnbl — DISMISSED** [review]
I really like the functional addition of resetting motors to 0 position.

**Comment by gwarby** [comment]
All 4 autonomous programs should have/use timesave functions, and arm motor reset feature

**Pull request merged**